version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
 // Version is the version of the build.
-const Version = "1.7.1-dev"
+const Version = "1.8.1-dev"
```

Comment:
I don't think dev should still be here, Just 1.8.0  @mtrmac @vrothberg ?

Reply:
Ooops, I was looking at the last commit and skipped over the middle one.  Ignore me.